Example/Podfile

```diff
@@ -1,3 +1,5 @@
+xcodeproj 'PINRemoteImage.xcodeproj'
```

Comment:
I initially built this with CocoaPods 0.39.0 and that version required this line. The current version for the project, 0.38.2, doesn't require it but also doesn't mind it so I left it. I can remove if needed.